core/src/main/java/bisq/core/user/Preferences.java

```diff
@@ -907,7 +911,7 @@ public boolean getPayFeeInBtc() {
 
     public long getWithdrawalTxFeeInVbytes() {
         return Math.max(prefPayload.getWithdrawalTxFeeInVbytes(),
-                Config.baseCurrencyNetwork().getDefaultMinFeePerVbyte());
```

Comment:
Will `BaseCurrencyNetwork.getDefaultMinFeePerVbyte()` be `@Deprecated` and/or removed?

Reply:
We still use `BaseCurrencyNetwork.getDefaultMinFeePerVbyte()` in the following cases: (a) when Bisq has started up before the first API call to PriceNode completes, (b) if the PriceNode is unreachable, and (c) if PriceNode does not provide the new field in the JSON.

Reply:
I was too lazy to check myself, so thanks for that. (resolved)